Source/UnitEx.h

```diff
@@ -28,9 +28,11 @@ class UnitEx : public Unit
 	CommandType GetLastCommand();
 	ActionType GetCurAction();
 	int CreatorID();
-	int IsEMPedEx();
-	int IsStickyfoamed();
-	int IsESGed();
+	int GetTimeEMPed();

```

Comment:
Should these functions return shorts since the Timer is stored as a short? 

If there was ever a set of SetTime functions, they would need to ingest a short.

Reply:
Hmm, good question. I'm fine with these returning `int`. Any calculations involving them will likely be done at `int` precision. If we create a corresponding set method, it could take either a `short` or an `int`. Either way, we may need to range check it, and potentially throw an exception for bad values. In particular, negative values might still be a problem, or if bitfields are used somewhere, with fewer than a full 16-bit short.